.travis.yml

```diff
@@ -4,12 +4,21 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
 
 env:
   - SYMFONY_VERSION=2.8.*
   - SYMFONY_VERSION=3.1.*
+  - SYMFONY_VERSION=4.0.*
+
+matrix:
+    fast_finish: true
+    include:
+        - php: 7.1
+          env: DEPENDENCIES=beta
```

Comment:
You should explicitly add `SYMFONY_VERSION=4.0.*` to this line and remove it from the `env` matrix above: this avoids a single failing build due to composer wanting to install Symfony 4.0.* without `minimum-stability: beta`. This can then be changed to be like all other version constraints once Symfony 4 is released.